kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -96,6 +96,35 @@
   const UserKinds = require('kolibri.coreVue.vuex.constants').UserKinds;
 
   module.exports = {
+    $trNameSpace: 'user-edit-modal',
+    $trs: {
+      modalTitle: 'Edit Account Info',
+      // input labels
+      fullName: 'Full Name',
+      username: 'Username',
+      userKind: 'User Kind',
+      enterNewPw: 'Enter new password',
+      confirmNewPw: 'Confirm new password',
```

Comment:
I don't see any problem in adopting sentence case (in truth, that has been be my personal preference since... ever) as per Material Design convention, but that would mean that all those strings which are written in title case now, should be corrected. 

If there's one constant fact in all the discussions I've seen on this topic, it is that which ever convention you pick, you should stick with it and be consistent **everywhere**. According to MD that's `all titles, headings, labels, menu items`. So:

`Edit account info`
`Full name`
`User kind/role/type`
`Delete user`
`Reset password`
etc.